plugins/GSdx/GSOsdManager.cpp

```diff
@@ -187,97 +212,112 @@ void GSOsdManager::RenderString(GSVertexPT1* dst, std::u32string msg, float x, f
 size_t GSOsdManager::Size(float m_sx, float m_sy) {
 	size_t sum = 0;
 
-	for(const auto &pair : m_monitor) {
-		sum += pair.first.size();
-		sum += pair.second.size();
-	}
+	// This does not give an accurate size, but it will be bigger or equal to what we need
+	if(theApp.GetConfigB("log_enabled")) {
```

Comment:
I don't know if code is often call but `theApp.GetConfig*` is very slow. I would suggest to cache it in a member variable. (same remark for the calls below)